# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../uclibc/pkg_patch/man-index.patch
# Copyright (C) 2006 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

--- ./src/to_cat.c.orig	2006-12-14 23:02:21.000000000 -0300
+++ ./src/to_cat.c	2006-12-14 23:02:29.000000000 -0300
@@ -2,8 +2,6 @@
 #include <stdlib.h>
 #include <string.h>
 
-extern char *rindex (const char *, int);	/* not always in <string.h> */
-
 #include "defs.h"
 #include "manfile.h"
 #include "man-config.h"
--- ./src/man.c.orig	2006-12-14 23:01:00.000000000 -0300
+++ ./src/man.c	2006-12-14 23:01:11.000000000 -0300
@@ -38,9 +38,6 @@
 #define R_OK 4
 #endif
 
-extern char *index (const char *, int);		/* not always in <string.h> */
-extern char *rindex (const char *, int);	/* not always in <string.h> */
-
 #include "defs.h"
 #include "gripes.h"
 #include "man.h"
--- ./src/man-config.c.orig	2006-12-14 23:02:02.000000000 -0300
+++ ./src/man-config.c	2006-12-14 23:02:10.000000000 -0300
@@ -35,8 +35,6 @@
 
 #define BUFSIZE 4096
 
-extern char *rindex (const char *, int);	/* not always in <string.h> */
-
 #define whitespace(x) ((x) == ' ' || (x) == '\t')
 
 /* directories listed in config file */
--- ./src/manpath.c.orig	2006-12-14 23:01:32.000000000 -0300
+++ ./src/manpath.c	2006-12-14 23:01:45.000000000 -0300
@@ -27,10 +27,6 @@
 #include <sys/types.h>
 #include <sys/stat.h>
 
-/* not always in <string.h> */
-extern char *index(const char *, int);
-extern char *rindex(const char *, int);
-
 #include "defs.h"
 #include "gripes.h"
 #include "man.h"		/* for debug */
